scripts/o3de/ExportScripts/export_source_android.py

```diff
@@ -124,7 +124,7 @@ def export_source_android_project(ctx: exp.O3DEScriptExportContext,
         except FileNotFoundError:
             logger.error("Unable to open build.gradle file. Aborting deployment...")
     
-    logger.info(f"Exporting finished. Output Android Project generated at {target_android_project_path}")
+    logger.info(f"Exporting finished. Output Android Project generated at '{target_android_project_path}'")
```

Comment:
Lets make the log of the output folder consistent for all platforms. No need to say what type of platform it is.
```suggestion
    logger.info(f"Exporting finished. Output Android Project generated at '{target_android_project_path}'")
```